### ISSUE TITLE
Give Option to Set "Open In New Window" as default target for links

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#149]
```
it's kinda annoying to have to keep clicking this option for each link when I 
want all my links to open in a new window. Firefox version had this, would be 
nice in Chrome to.
```

-----
Original issue reported on code.google.com by `tkriegls...@gmail.com` on 31 Aug 2010 at 5:30

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `cfinke` on 12 Sep 2010 at 1:57
* Changed title: **Give Option to Set "Open In New Window" as default target for links**
* Changed state: **Accepted**
* Added labels: **Type-Enhancement**, **Priority-Low**

**Comment by GoogleCodeExporter** [comment]
```
Done in r319; it will be included in version 1.9.
```

-----
Original comment by `cfinke` on 22 Aug 2011 at 3:16
* Changed state: **Fixed**
* Added labels: **Milestone-1.9**
